README.md

```diff
@@ -422,6 +422,11 @@ your development environment is likely a good starting point:
     cp .env .env.integration
 
 
+There is also a cleanup routine intended for use by CI services to check for and cleanup any dangling OpenStack VMs past a certain age threshold. While it isn't necessary in the usual case, old integration tests that were killed without cleanup can be cleaned up after four hours by running the make target:
+
+    make test_integration_cleanup
+
+
```

Comment:
@bdero Thanks for the documentation update. It might be worth adding a note about adjusting the age threshold by setting `INSTANCE_AGE_THRESHOLD` in `.env.integration`. And possibly a warning about accidentally terminating instances that are in use by running CircleCI builds (if integration tests running locally are using the same OpenStack account as CircleCI).